src/System.Security.SecureString/src/System/Security/SecureStringMarshal.CoreCLR.cs

```diff
@@ -4,14 +4,12 @@
 
 namespace System.Security
 {
+	[System.CLSCompliant(false)]
     public static class SecureStringMarshal
     {
-        public static IntPtr SecureStringToCoTaskMemAnsi(SecureString s) => s != null ? s.MarshalToString(globalAlloc: false, unicode: false) : IntPtr.Zero;
-
-        public static IntPtr SecureStringToGlobalAllocAnsi(SecureString s) => s != null ? s.MarshalToString(globalAlloc: true, unicode: false) : IntPtr.Zero;
-
-        public static IntPtr SecureStringToCoTaskMemUnicode(SecureString s) => s != null ? s.MarshalToString(globalAlloc: false, unicode: true) : IntPtr.Zero;
-
-        public static IntPtr SecureStringToGlobalAllocUnicode(SecureString s) => s != null ? s.MarshalToString(globalAlloc: true, unicode: true) : IntPtr.Zero;
+        public static IntPtr SecureStringToCoTaskMemAnsi(SecureString s) => IntPtr.Zero;
+        public static IntPtr SecureStringToGlobalAllocAnsi(SecureString s) => IntPtr.Zero;
+        public static IntPtr SecureStringToCoTaskMemUnicode(SecureString s) => IntPtr.Zero;
+        public static IntPtr SecureStringToGlobalAllocUnicode(SecureString s) => IntPtr.Zero;
```

Comment:
SecureString is now implemented in CoreLib and the methods that would do that conversion are internal.

They are exposed on the Marshal class with this PR: https://github.com/dotnet/coreclr/pull/7985 and there is an issue to move these APIs to use the Marshal APIs once the new CoreCLR is ready.https://github.com/dotnet/corefx/issues/13331